infra/cifuzz/cifuzz.py

```diff
@@ -203,6 +209,109 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   return True, False
 
 
+def get_project_coverage_report(project_name):
+  """Gets the report json for a specific OSS-Fuzz project from GCS.
+
+  Args:
+    project_name: The name of the relevant OSS-Fuzz project.
+
+  Returns:
+    The projects coverage json dict or None on failure.
+  """
+  project_report_url = fuzz_target.url_join(fuzz_target.GCS_BASE_URL,
+                                            PROJECT_COVERAGE_REPORT,
+                                            project_name + '.json')
+  project_cov_json = get_json_from_url(project_report_url)
+  if not project_cov_json:
+    logging.error('Could not get the coverage report json from %s.',
+                  project_report_url)
+    return None
+  if 'fuzzer_stats_dir' not in project_cov_json:
+    logging.error('fuzzer_stats_dir is not a key in downloaded json.')
+    return None
+  return project_cov_json
+
+
+def get_target_coverage_report(project_cov_report, target_name):
+  """Get the coverage report from a specific fuzz target.
+
+  Args:
+    project_cov_report: A dict containing the project cov report data.
+    target_name: The name of the fuzz target whos coverage is requested.
+
+  Returns:
+    The targets coverage json dict or None on failure.
+  """
+  if 'fuzzer_stats_dir' not in project_cov_report:
+    return None
+  fuzzer_report_url_segment = project_cov_report['fuzzer_stats_dir']
+
+  # Converting gs:// to http://
+  fuzzer_report_url_segment = fuzzer_report_url_segment.replace('gs://', '')
+  target_url = fuzz_target.url_join(fuzz_target.GCS_BASE_URL,
+                                    fuzzer_report_url_segment,
+                                    target_name + '.json')
+  return get_json_from_url(target_url)
+
+
+def get_files_covered_by_target(project_cov_report, target_name,
+                                oss_fuzz_project_base):
+  """Gets a list of files covered by the specific fuzz target.
+
+  Args:
+    project_cov_report: A dict containing the project cov report data.
+    target_name: The name of the fuzz target whose coverage is requested.
+    oss_fuzz_project_base: The location where OSS-Fuzz project is cloned too for
+      the projects build.
+
+  Returns:
+    A list of files that the fuzzer covers or None.
+  """
+  if not oss_fuzz_project_base:
+    return None
+  target_cov = get_target_coverage_report(project_cov_report, target_name)
+  if not target_cov:
+    return None
+  if not target_cov['data'][0]['files']:
```

Comment:
assign this to a local variable e.g. `files` or `coverage_per_file`, and re-use it in the check plus on the line 283

Reply:
Done.